ibm/mas_devops/roles/ocp_provision/defaults/main.yml

```diff
@@ -88,3 +89,7 @@ ipi_config_dir: "{{ ipi_dir }}/config/{{ cluster_name }}"
 
 ocp_installer_dir: "{{ ipi_dir }}/installer/{{ ocp_version }}"
 ocp_installer_exe: "{{ ipi_dir }}/installer/{{ ocp_version }}/openshift-install"
+
+# Dualstack required?
+ipv6_test: "{{ lookup('env', 'IPV6_TEST') | default('false', true) }}"
+fyre_root_password: "{{ lookup('env', 'FYRE_ROOT_PASSWORD') | default('false', true) }}"
```

Comment:
I don't think we use `fyre_root_password` anywhere do we?

Reply:
It doesn't look like fyre_root_password is used. Let me remove that and retest